src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift

```diff
@@ -171,7 +171,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 		updateExposureState(state)
 		Analytics.triggerAnalyticsSubmission()
 		appUpdateChecker.checkAppVersionDialog(for: window?.rootViewController)
-		healthCertificateService.checkIfBoosterRulesShouldBeFetched()
+		healthCertificateService.checkIfBoosterRulesShouldBeFetched(completion: {_ in })
```

Comment:
Can we do it more beautiful, like passing a default value or something like that?

Reply:
yeah we can do that I can log the error